frontend/src/scenes/insights/insightLogic.ts

```diff
@@ -380,6 +380,10 @@ export const insightLogic = kea<insightLogicType>([
             (props: InsightLogicProps) =>
                 !!props.dashboardItemId && props.dashboardItemId !== 'new' && !props.dashboardItemId.startsWith('new-'),
         ],
+        isInExperimentContext: [
```

Comment:
wonder if it's better to figure this out by passing in a parameter to InsightLogicProps instead, would be annoying if this breaks due to URL updates / breaks unexpectedly in other places

Reply:
But if it's short term, probably not that big a deal